index.js

```diff
@@ -103,6 +112,7 @@ function split (matcher, mapper, options) {
   stream.matcher = matcher
   stream.mapper = mapper
   stream.maxLength = options.maxLength
+  stream.skipOverflow = options.skipOverflow
```

Comment:
Can you add a `stream.overflow` property here?

Reply:
I will do that in the morning.